nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/SNMPWorker.java

```diff
@@ -47,7 +46,7 @@
      * @param snmp instance of {@link Snmp}
      * @param target instance of {@link AbstractTarget}
      */
-    public SNMPWorker(Snmp snmp, AbstractTarget target) {
+    protected SNMPWorker(Snmp snmp, AbstractTarget target) {
```

Comment:
I noticed a few public->private changes as well throughout the PR. While I can't argue that the access modifier you went with would have been the best initial choice, making a change back from public can break other people's code, even if it isn't in NiFi now (in other words, the changes are a public API change).